components/netlify/netlify.app.js

```diff
@@ -0,0 +1,155 @@
+const axios = require("axios");
+const crypto = require("crypto");
+const jwt = require('jwt-simple');
+const NetlifyAPI = require("netlify");
+const parseLinkHeader = require('parse-link-header');
+
+module.exports = {
+  type: "app",
+  app: "netlify",
+  propDefinitions: {
+    siteId: {
+      type: "string",
+      label: "Site ID",
+      description: "The site for which events must be captured.",
```

Comment:
Small style nit: remove the period at the end of descriptions (unless you're using full sentences)

Reply:
Done!